Sources/Sandbox.Game/Game/Entities/Blocks/MyOreDetector.cs

```diff
@@ -167,9 +167,9 @@ public override void UpdateBeforeSimulation100()
                 m_oreDetectorComponent.Clear();
             }
 
-            if (ticksSinceOreMarkersCalled < Stopwatch.Frequency)
+            if (MethodCallRateCounter < Stopwatch.Frequency)
             {
-                ticksSinceOreMarkersCalled++;
+                MethodCallRateCounter++;
```

Comment:
Because than no duplicate of the variable is create that is returned at this point.

Like this:
`varType incrementBeforeReturn() 
{
  var = var +1;
  return var;
}`

`varType incrementAfterReturn()
{
  varType prev = var;
  var = var +1;
  return prev;
}`

incrementing before before return therefore reduces memory consumption and runtime